src/main/scala/lenthall/spray/SprayCanHttpService.scala

```diff
@@ -75,8 +75,8 @@ object SprayCanHttpService {
     : Future[Unbound] = {
       unbind(duration) andThen {
         case _ =>
-          actorSystem.shutdown()
-          actorSystem.awaitTermination()
+          actorSystem.terminate()
+          Await.ready(actorSystem.whenTerminated, Duration.Inf)
```

Comment:
Didn't mean for that to sound so curt. Will dig up a reference when I'm not on my phone